addons/payment_stripe_sca/models/payment.py

```diff
@@ -58,7 +58,7 @@ def _stripe_request(self, url, data=False, method="POST"):
         # cfr https://stripe.com/docs/error-codes
         # these can be made customer-facing, as they usually indicate a problem with the payment
         # (e.g. insufficient funds, expired card, etc.)
-        if not resp.ok and not (400 <= resp.status_code < 500 and resp.json().get('error', {}).get('code')):
+        if not resp.ok and self._context.get('raise_exception') and (400 <= resp.status_code < 500 and resp.json().get('error', {}).get('code')):
```

Comment:
I think `raise_exception` is too generic as a context key, the key should be named to something more specific to Stripe.

Otherwise I think this will fix the issue :)

Reply:
@tbe-odoo TY
This one should be unique.
It can go up to saas-12.3
I'll do a separate PR for 13.0 and up